sound/audioclass.py

```diff
@@ -95,34 +55,27 @@ def GetAudioClass(self, data):
         else:
             d.popleft();d.append(self.Decibel)
             self.loop += 1
+        change = data.suddenchange
+        print "sudden sound change value {}".format(data.suddenchange)
+    	
+        loud = '(StateLink Sudden sound change value (NumberNode "' + \
```

Comment:
Anyway, this is all merged now.  We will test this tomorrow.  It would be great if you could create a pull req to HEAD that adds a ros node for luminance -- one that would be very much like the one for sound, that was already merged.  (the ros node for luminance would not use any blender or opencog, or anything -- just a very very simple brightness ROS node)